app/models/concerns/navigation/siblings_navigation.rb

```diff
@@ -8,6 +8,14 @@ def restart(user)
     pending_siblings_for(user).sort_by(&:number).first
   end
 
+  def siblings
+    structural_parent.structural_children
+  end
+
+  def pending_siblings_for(user, organization=Organization.current)
```

Comment:
Are se planning to change this method when indicators table is implemented?

I am wondering whether this query will become too heavy now

Reply:
It does use indicators table.
We are retrieving the associated progress ítem for each content.
In higher levels of the tree it could be very expensive in case of too many dirty indicators for that user.
In a normal flow, that's very rare though.